test/messages.test.js

```diff
@@ -19,199 +16,196 @@ const metafeedKeys = keys.deriveFeedKeyFromSeed(
   'metafeed',
   'bendybutt-v1'
 )
+const mainKey = ssbKeys.generate()
 
-const dir = '/tmp/metafeeds-messages'
-const mainKey = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
-
-rimraf.sync(dir)
-
-let sbot = SecretStack({ appKey: caps.shs })
-  .use(require('ssb-db2'))
-  .use(require('ssb-bendy-butt'))
-  .use(require('../'))
-  .call(null, {
-    keys: mainKey,
-    path: dir,
-  })
-let db = sbot.db
-let messages = sbot.metafeeds.messages
-
-let addMsg
-
-test('add a feed to metafeed', (t) => {
-  const opts = messages.optsForAddExisting(metafeedKeys, 'main', mainKey)
-
-  t.equal(opts.content.subfeed, mainKey.id, 'correct subfeed id')
-  t.notOk(opts.content.nonce, 'should have no nonce')
-  t.equal(opts.content.metafeed, metafeedKeys.id, 'correct metafeed id')
-
-  db.create(opts, (err, kvt) => {
-    t.error(err, 'no error')
-    addMsg = kvt
-    t.end()
-  })
-})
+test('messages', (t) => {
+  const sbot = Testbot({ keys: mainKey })
+  const {
+    db,
+    metafeeds: { messages },
+  } = sbot
 
-let tombstoneMsg
+  let addMsg
+  // NOTE - these subtests are not "atomic" - db state is shared between them
```

Comment:
This diff is annoying. I didn't touch the tests at all other then indenting them, and changing to `t.test`

Note the shared variables and sbot are now "inside" a test scope (not global in the file)